receiver/opencensusreceiver/config.go

```diff
@@ -44,6 +44,8 @@ type Config struct {
 
 // tlsCredentials holds the fields for TLS credentials
 // that are used for starting a server.
+// TODO(ccaraman): Add validation to check that these files exist at configuration loading time.
```

Comment:
Should I assing https://github.com/open-telemetry/opentelemetry-service/issues/170 to you?

Reply:
Yes please! Ps. I really appreciate that issues that you've filled!